### PR TITLE
Make leaf filenames consistent, update opaque textures

### PR DESCRIPTION
**Pull request by careeoki**
In prep for #482 . Also, the hue of pine needles is slightly more green now.

### CODE REVIEW

File: assets/cubyz/blocks/oak_leaves.json [hunk 1]
```diff
@@ -9,5 +9,5 @@
 	"alwaysViewThrough" : true,
 	"absorbedLight" : 0x121012,
 	"model" : "cube",
-	"texture" : "cubyz:oak_leaves_transparent"
+	"texture" : "cubyz:oak_leaves"
```
Comment (IntegratedQuantum) [inline assets/cubyz/blocks/oak_leaves.json:12]: You also need to change that for the non_degradable_oak_leaves

**Comment by careeoki** [comment]
Oh, right. I also fixed some other stuff.
Would leaf blocks be able to be merged into one block in the future?

**Comment by IntegratedQuantum** [comment]
The non_degradable leaves is really just a hack to get the foggy forest generating correctly.
They will be removed completely once that situation is sorted out. There is also no need to add non_degradable pine needles, since there is no foggy biome for them currently.

**Comment by IntegratedQuantum** [comment]
I have decided to merge this without the non_degradable pine needles.

**Comment by careeoki** [comment]
> The non_degradable leaves is really just a hack to get the foggy forest generating correctly. 
Ohhh, that’s what they're for.

**Pull request merged**